### PR TITLE
25 validation

### PR DESCRIPTION
**Pull request by JustDaile**
Removed validation library and added some basic validation. This can be vastly improved in the future.

### CODE REVIEW

File: pkg/net/dtos/google.go
Comment (ghost) [inline pkg/net/dtos/google.go:1]: please remove the `validate` tags on both `GoogleSignUpRequest` and `GoogleSignInRequest` as they were being used by the validator library.

**Review by ghost — COMMENTED** [review]
everything else looks good to me.

File: pkg/net/dtos/google.go
Comment (JustDaile) [reply pkg/net/dtos/google.go:1]: Done

**Review by ghost — APPROVED** [review]
lgtm

**Pull request merged**